src/hisp/festim_models/mb_model.py

```diff
@@ -247,21 +251,289 @@ def make_W_mb_model(
+            F.VTXSpeciesExport(f"{folder}/trapped_concentration_d1.bp", field=trap1_D),
+            F.VTXSpeciesExport(f"{folder}/trapped_concentration_t1.bp", field=trap1_T),
+            F.VTXSpeciesExport(f"{folder}/trapped_concentration_d2.bp", field=trap2_D),
+            F.VTXSpeciesExport(f"{folder}/trapped_concentration_t2.bp", field=trap2_T),
+            F.VTXSpeciesExport(f"{folder}/trapped_concentration_d3.bp", field=trap3_D),
+            F.VTXSpeciesExport(f"{folder}/trapped_concentration_t3.bp", field=trap3_T),
+        ]
+
+    quantities = {}
+    for species in my_model.species:
+        quantity = F.TotalVolume(field=species, volume=w_subdomain)
+        my_model.exports.append(quantity)
+        quantities[species.name] = quantity
+
+    ############# Settings #############
+    my_model.settings = F.Settings(
+        atol=1e-10,
+        rtol=1e-10,
+        max_iterations=1000000,
+        final_time=final_time,
+    )
+
+    my_model.settings.stepsize = F.Stepsize(initial_value=1)
+
+    return my_model, quantities
+
+
+def make_B_mb_model(
+    temperature: Callable | float | int,
+    deuterium_ion_flux: Callable,
+    tritium_ion_flux: Callable,
+    deuterium_atom_flux: Callable,
+    tritium_atom_flux: Callable,
+    final_time: float,
+    folder: str,
+    L: float,
+    exports=False,
+) -> Tuple[CustomProblem, Dict[str, F.TotalVolume]]:
+    """Create a FESTIM model for the B MB scenario.
+
+    Args:
+        temperature: the temperature in K.
+        deuterium_ion_flux: the deuterium ion flux in m^-2 s^-1.
+        tritium_ion_flux: the tritium ion flux in m^-2 s^-1.
+        deuterium_atom_flux: the deuterium atom flux in m^-2 s^-1.
+        tritium_atom_flux: the tritium atom flux in m^-2 s^-1.
+        final_time: the final time in s.
+        folder: the folder to save the results.
+        L: the length of the domain in m.
+
+    Returns:
+        the FESTIM model, the quantities to export.
+    """
+    my_model = CustomProblem()
 
-    my_model.exports = [
-        F.VTXSpeciesExport(f"{folder}/mobile_concentration_t.bp", field=mobile_T),
-        F.VTXSpeciesExport(f"{folder}/mobile_concentration_d.bp", field=mobile_D),
-        F.VTXSpeciesExport(f"{folder}/trapped_concentration_d1.bp", field=trap1_D),
-        F.VTXSpeciesExport(f"{folder}/trapped_concentration_t1.bp", field=trap1_T),
-        F.VTXSpeciesExport(f"{folder}/trapped_concentration_d2.bp", field=trap2_D),
-        F.VTXSpeciesExport(f"{folder}/trapped_concentration_t2.bp", field=trap2_T),
-        F.VTXSpeciesExport(f"{folder}/trapped_concentration_d3.bp", field=trap3_D),
-        F.VTXSpeciesExport(f"{folder}/trapped_concentration_t3.bp", field=trap3_T),
+    ############# Material Parameters #############
+
+    vertices = np.concatenate(  # 1D mesh with extra refinement
+        [
+            np.linspace(0, 30e-9, num=200),
+            np.linspace(30e-9, L, num=200),
+        ]
+    )
```

Comment:
Note to self:

For all models, for all conditions, the mesh is always the same. 30 nm of local refinement. 
Do we have any sense for if it's valid for the range of exposure conditions? Is the rough refinement (ie. from 30 nm to L) enough?